apps/aecore/test/aec_tx_pool_tests.erl

```diff
@@ -96,6 +96,87 @@ tx_pool_test_() ->
                %% No single tx would have fitted on top of this
                ?assert(MinGas > aec_governance:block_gas_limit() - TotalGas)
        end},
+      {"fill micro block with and without previously rejected tx",
+       fun() ->
+               aec_test_utils:stop_chain_db(),
+               {ok, MinerPubKey} = aec_keys:pubkey(),
+               PubKey1 = new_pubkey(),
+               PubKey2 = new_pubkey(),
+               meck:expect(aec_genesis_block_settings, preset_accounts, 0,
+                           [{PubKey1, 20001}, {PubKey2, 20000000}]),
+               {Block0, _} = aec_block_genesis:genesis_block_with_state(),
+               aec_test_utils:start_chain_db(),
```

Comment:
I'm not sure why we need to stop and start chain db here?

Reply:
It's a common pattern across our test codebase to initialize chain with custom preset accounts. I could mine instead but it's quicker and cleaner this way